src/main/java/org/ibp/api/rest/germplasmlist/GermplasmListResourceGroup.java

```diff
@@ -149,7 +150,20 @@
 		@RequestParam(required = false) final String programUUID,
 		@RequestBody final SearchCompositeDto<GermplasmListDataSearchRequest, Integer> searchComposite
 	) {
-		this.germplasmListService.addGermplasmListEntriesToAnotherList(crop, programUUID, germplasmListId, sourceGermplasmListId, searchComposite);
+		this.germplasmListService.addGermplasmListEntriesToAnotherList(crop, programUUID, germplasmListId, sourceGermplasmListId,
+			searchComposite);
+		return new ResponseEntity<>(HttpStatus.OK);
+	}
+
+	@ApiOperation(value = "Remove germplasm entries from an existing list")
+	@RequestMapping(value = "/crops/{crop}/germplasm-lists/{germplasmListId}/entries", method = RequestMethod.DELETE)
+	@PreAuthorize("hasAnyAuthority('ADMIN', 'GERMPLASM', 'MANAGE_GERMPLASM', 'GERMPLASM_LISTS', 'MANAGE_GERMPLASM_LISTS', 'REMOVE_ENTRIES_GERMPLASM_LISTS')")
+	public ResponseEntity<Void> removeGermplasmEntriesFromList(
```

Comment:
Use @RequestBody final` SearchCompositeDto<GermplasmListDataSearchRequest, Integer>` searchComposite instead of `List<Integer>`